src/EnergyPlus/HybridModel.cc

```diff
@@ -142,35 +165,222 @@ namespace HybridModel {
-                        ShowContinueError("Field " + cAlphaFieldNames(4) + "\" is changed to NO for the hybrid modeling simulations.");
+                    TemperatureSchPtr = GetScheduleIndex(cAlphaArgs(6));
+                    HumidityRatioSchPtr = GetScheduleIndex(cAlphaArgs(7));
+                    CO2ConcentrationSchPtr = GetScheduleIndex(cAlphaArgs(8));
+
+                    // Not used for now
+                    PeopleActivityLevelSchPtr = GetScheduleIndex(cAlphaArgs(9));
+                    PeopleSensibleFractionSchPtr = GetScheduleIndex(cAlphaArgs(10));
+                    PeopleRadiantFractionSchPtr = GetScheduleIndex(cAlphaArgs(11));
+                    PeopleCO2GenRateSchPtr = GetScheduleIndex(cAlphaArgs(12));
+
+                    // Pointers used to help decide wheather to include system supply terms in the inverse algorithms
+                    SupplyAirTemperatureSchPtr = GetScheduleIndex(cAlphaArgs(13));
+                    SupplyAirMassFlowRateSchPtr = GetScheduleIndex(cAlphaArgs(14));
+                    SupplyAirHumidityRatioSchPtr = GetScheduleIndex(cAlphaArgs(15));
+                    SupplyAirCO2ConcentrationSchPtr = GetScheduleIndex(cAlphaArgs(16));
+
+                    /*  Note: Internal thermal mass can be calculated only with measured temperature.
+                                      Air infiltration rate can be calculated with either measured temperature, humifity ratio, or CO2 concentration.
+                                      People count can be calculated with either measured temperature, humifity ratio, or CO2 concentration.
+                    */
+                    // Initially set all flags to be false
+                    HybridModelZone(ZonePtr).InternalThermalMassCalc_T = false;
+                    HybridModelZone(ZonePtr).InfiltrationCalc_T = false;
+                    HybridModelZone(ZonePtr).InfiltrationCalc_H = false;
+                    HybridModelZone(ZonePtr).InfiltrationCalc_C = false;
+                    HybridModelZone(ZonePtr).PeopelCountCalc_T = false;
+                    HybridModelZone(ZonePtr).PeopelCountCalc_H = false;
+                    HybridModelZone(ZonePtr).PeopelCountCalc_C = false;
+
+                    Flag_InternalThermalMassCalc = false;
+                    Flag_AirInfiltrationCalc = false;
+                    Flag_PeopleCountCalc = false;
+
+                    // Scenario 1: Only one unknown parameter to solve
+                    // Scenario 1-1: To solve thermal mass
+                    if (FlagHybridModel_TM) {
+                        if (FlagHybridModel_AI) {
+                            ShowContinueError("Field \"" + cAlphaFieldNames(3) + " and " + cAlphaFieldNames(4) + "\" cannot be both set to YES.");
+                            ShowContinueError("Field \"" + cAlphaFieldNames(4) + "\" is changed to NO for the hybrid modeling simulations.");
+                        }
```

Comment:
These ShowContinueError does not have a ShowSevereError.
ShowSevereError raises an Exception to the program. ShowContinueError just adds up to the logging message.

Reply:
Same as follows.

Reply:
In this case, when both "calculate thermal mass" and "calculate infiltration" are set to YES, the code should calculate the thermal mass instead of the other. I'm not familiar with the normal practice. Should we terminate the process with the severe error, or continue the simulation but give a error message?

Reply:
@tsbyq If you feel like the program should be terminated, the routine should be
ShowSevereError("blahblah");
ShowContinueError("continue blahblah");
ShowContinueError("continue blahblah");
...

If not (even if the input is not valid, the error is tolerable - such as imported schedule not used), the routine should be:
ShowWarningError("blahblah");
ShowContinueError("continue blahblah");
ShowContinueError("continue blahblah");
...